src/main/java/crazypants/enderzoo/spawn/MobSpawnEventHandler.java

```diff
@@ -213,12 +214,12 @@ public void onBlockHarvest(HarvestDropsEvent event) {
 
       EntityDireSlime direSlime = new EntityDireSlime(event.getWorld());
       direSlime.setPosition(event.getPos().getX() + 0.5, event.getPos().getY() + 0.0, event.getPos().getZ() + 0.5);
-      event.getWorld().spawnEntityInWorld(direSlime);
+      event.getWorld().spawnEntity(direSlime);
       direSlime.playLivingSound();
       for (ItemStack drop : event.getDrops()) {
         if (drop != null && drop.getItem() != null && drop.getItem() == Item.getItemFromBlock(Blocks.DIRT)) {
-          if (drop.stackSize > 1) {
-            drop.stackSize--;
+          if (drop.getCount() > 1) {
+            drop.shrink(1);
           } else if (event.getDrops().size() == 1) {
             event.getDrops().clear();
           } else {
```

Comment:
in isToolEffective(), line 235, you missed a stack null-check. (sry, cannot annotate there)